.github/workflows/tox.yml

```diff
@@ -229,3 +228,42 @@ jobs:
         uses: codecov/codecov-action@v4
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
+
+      - if: ${{ (success() || failure()) && contains(matrix.coverage, 'github') && matrix.pytest == 'true' }}
+        run: |
+          mv .coverage .coverage.${{ runner.os }}-${{ runner.arch }}-${{ matrix.toxenv }}
```

Comment:
So, this step appears to use `runs-on: ${{ matrix.os }}` above. What if someone tries to do this coverage stuff on Windows?

Reply:
`runs-on` accepts the name of the Docker image (most commonly `ubuntu-latest`, `macos-latest`, or `windows-latest`), but we don't necessarily want that in the coverage combination; `runner.arch` will be `Linux`, `macOS`, or `Windows`

Reply:
`mv` command doesn't exist on Windows, does it?

Reply:
it does as an alias in powershell, which I believe is the default for Windows runners:
https://learn.microsoft.com/en-us/powershell/module/Microsoft.PowerShell.Management/move-item?view=powershell-7.4#notes

However to be safe I can set the shell to `bash` here